posts/2022-12-05-survey-launch.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Launching the 2022 State of Rust Survey"
+author: The Rust Survey Team
+description: "Hearing from you about the seventh year of Rust"
+---
+
+It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Survey Team is pleased to announce our [2022 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and establish development priorities for the future.
```

Comment:
I think it would be a good idea to call out the survey and link to it immediately like this:

> ### The [2022 State of the Rust Survey](link) is here!

> It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Survey Team is pleased to announce our [2022 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the Project understand its strengths and weaknesses, and establish development priorities for the future.